app/workers/bgs/submit_form686c_job.rb

```diff
@@ -19,7 +19,14 @@ def perform(user_uuid, saved_claim_id, vet_info)
       claim_data = valid_claim_data(saved_claim_id, vet_info)
 
       BGS::Form686c.new(user).submit(claim_data)
+
+      # If Form 686c job succeeds, then enqueue 674 job.
+      claim = SavedClaim::DependencyClaim.find(saved_claim_id)
+
+      BGS::SubmitForm674Job.perform_async(user.uuid, claim.id, vet_info) if claim.submittable_674?
```

Comment:
Since you already have some of these values I MAY have done this instead:

```
BGS::SubmitForm674Job.perform_async(user_uuid, saved_claim_id, vet_info) if claim.submittable_674?
```
To avoid 2 extra calls, but it wont make a difference anyways I dont think, like it is not a high-volume thing happening a lot that that perf improvement would matter, but just a thought.

Reply:
Oh, nice catch! I'll update that now.